src/libraries/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -485,6 +485,19 @@ public bool IsIPv6Teredo
             }
         }
 
+        /// <devdoc>
+        ///   <para>
+        ///     Determines if an address is an IPv6 Unique Local address
+        ///   </para>
+        /// </devdoc>
```

Comment:
```suggestion
        /// <summary>Gets whether the address is an IPv6 Unique Local address.</summary>
```

Reply:
Previously created implementations of such properties adhere to the documentation I have provided (for example, `IsIPv6LinkLocal`). Is it exactly necessary to replace this with summary?
If a replacement is necessary, then is it necessary to apply such a style to existing similar properties?

Reply:
You don't need to update the existing ones: we'll be taking care of that in the coming months. But yes, please use my suggestion instead. Thanks!

Reply:
Okay, thank you